src/routes/account.router.js

```diff
@@ -31,125 +31,126 @@ router.post('/create',
         res.status(200).send(acc);
       })
       .catch((err) => {
-        if (err instanceof Err.Account){
+        if (err instanceof Err.Account) {
           logger.account.error(`Failed Account Creation ${req.body.email}`);
           logger.account.error(`  Reason: ${err.message}`);
           res.status(err.status).send(err.message);
         }
-        else{
+        else {
           res.sendStatus(500);
         }
       });
   }
 );
 
 router.post('/login',
-    [
-      check('email').exists().isEmail(),
-      check('password').exists(),
-    ],
-    RequestValidationMiddleware.handleErrors,
-    (req, res) => {
-      return AccountService.validatePassword(req.body.email, req.body.password)
-        .then((acc) => {
-          return TokenService.createAccessToken(acc);
-        })
-        .then((tok) => {
-          logger.account.info(`Successful login for ${req.body.email}`)
-          res.status(200).send({
-            accessToken: tok,
-          });
-        })
-        .catch((err) => {
-          if (err instanceof Err.Account){
-            logger.account.info(`Failed login for ${req.body.email}`);
-            res.status(err.status).send(err.message);
-          }
-          else {
-            res.status(500).send(err);
-          }
+  [
+    check('email').exists().isEmail(),
+    check('password').exists(),
+  ],
+  RequestValidationMiddleware.handleErrors,
+  (req, res) => {
+    return AccountService.validatePassword(req.body.email, req.body.password)
+      .then((acc) => {
+        return TokenService.createAccessToken(acc);
+      })
+      .then((tok) => {
+        logger.account.info(`Successful login for ${req.body.email}`)
+        res.status(200).send({
+          accessToken: tok,
```

Comment:
If possible in the future we should avoid making whitespace changes, it saturates the PR changes making review less seamless, thus increasing chances of stale or rubber-stamping reviews. 😄

Reply:
Do you have any suggestions on how to avoid whitespace changes?